dll/win32/setupapi/install.c

```diff
@@ -714,50 +710,6 @@ static BOOL register_dlls_callback( HINF hinf, PCWSTR field, void *arg )
     return ret;
 }
 
-#ifdef __WINESRC__
```

Comment:
Please keep this code (turn it into an `#ifndef __REACTOS__` instead).

Reply:
Fixed.